doc/Developing/os/Settings.md

```diff
@@ -99,6 +99,10 @@ per-device in the webui and per os or globally in config.php. Usually,
 a poller module will not work if it's corresponding discovery module
 is not enabled.
 
+You should avoid setting these to false the OS definitions unless it has a
```

Comment:
Grammar

Reply:
Missed a word in there, and read past it in proof reading.  Funny how that works.